semgrep-core/src/core/ast/Visitor_AST.ml

```diff
@@ -96,6 +97,7 @@ let default_visitor =
         ());
     ksvalue = (fun (k, _) x -> k x);
     kargument = (fun (k, _) x -> k x);
+    klit = (fun (k, _) x -> k x);
```

Comment:
Technically this does change the default. If the tests pass then it looks like nothing has been relying on the previous behavior, so I think that's fine, just wanted to make sure it was intentional.

Reply:
Could you elaborate? I didn't intend to change the visitor behavior.